easybuild/easyconfigs/p/PyTorch/USER.md

```diff
@@ -2,12 +2,14 @@
 
 **BETA VERSION, problems may occur and may not be solved quickly.**
 
+## Module and wrapper scripts
+
 The PyTorch container is developed by AMD specifically for LUMI and contains the
 necessary parts to run PyTorch on LUMI, including the plugin needed for RCCL when
 doing distributed AI, and a suitable version of ROCm for the version of PyTorch.
 The apex, torchvision, torchdata, torchtext and torchaudio packages are also included.
 
-The EasyBuild installation with the EasyConfigs mentioned below will do three things:
+The EasyBuild installation with the EasyConfigs mentioned below will do three or four things:
 
 1.  It will copy the container to your own file space. We realise containers can be
```

Comment:
For the file space: There is a long path underneath EBU_USER_PREFIX but I don't think it makes sense to specify that as it is complicated, but I agree it can be phrased better. The easy way to find the container for a user is to load the module and then there is the usual EBROOT* environment variable and to make it even easier I defined the equivalent CONTAINERROOT environment variable which is the same for all containers I offer through similar scripts which also makes it easier to then make generic documentation about, e.g., extending a container that provides Python with a virtual environment as that environment variable will exist for all those containers.